assets/stories/skate-components/detours/diversionPanel.stories.tsx

```diff
@@ -43,15 +42,6 @@ export const WithDirections: Story = {
 export const WithStops: Story = {
   args: {
     ...WithDirections.args,
-    missedStops: (
-      <>
-        <ListGroup variant="flush" as="ol">
-          <ListGroup.Item as="li">Stop 1</ListGroup.Item>
-          <ListGroup.Item as="li">Stop 2</ListGroup.Item>
-          <ListGroup.Item as="li">Stop 3</ListGroup.Item>
-          <ListGroup.Item as="li">Stop 4</ListGroup.Item>
-        </ListGroup>
-      </>
-    ),
+    missedStops: stopFactory.buildList(4),
```

Comment:
Are these deterministic?

Reply:
Good question - the sequences might not be depending on the order in which this runs relative to other factories. The thing that actually gets rendered is the name, so I think I'll explicitly specify the names so as to avoid spurious diffs in the future.

Reply:
Lol - yes they are

<img width="357" alt="Screenshot 2024-02-26 at 12 00 23 PM" src="https://github.com/mbta/skate/assets/912020/c3d6dfad-afaa-4d24-9f9b-11c072fc2540">

Reply:
ffe26ef0